README.md

```diff
@@ -11,7 +11,7 @@ This project is for educational purposes only. Pull request are welcome, but pri
 
 Site published at https://github.com/front-end-by-rimantas/20-grupe-portfolio-mitech
 
-Design: `Mitech` (https://demo.hasthemes.com/mitech-preview/index.html)
+Design: `Mitech` (https://demo.hasthemes.com/mitech-preview/index-infotechno.html)
```

Comment:
backtickai labiau skirti kodui pavaizduoti pvz console log('hello world')
kad link veiktu ant paspaudus ant teksto turi buti \[Textas](link)
galima paryskinti su **bold** --> bold

Reply:
Taip, standartinė sintaksė markdown'e yra `[text](href)`, tačiau kartais, jei norėtume atvaizduoti ir pačia nuorodą, tai galim be tokios sintaksės apsieiti - pasirinkimo laisvė 👍